### PR TITLE
Deploy plumbing Tekton resources to robocat

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Deploy resources under tekton/resources to the robocat cluster.

Work towards #224 
Depends on #276 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: robocat/README.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# Setting up the Robocat cluster from scratch
+
+These are step by step instructions on how to setup the `robocat` cluster using
+the available automation. The automation is based on Tekton, so it requires
+a cluster with Tekton deployed, which will run the tasks required to setup
```
Comment (dibyom) [inline robocat/README.md]:
```suggestion
a "driver" cluster with Tekton deployed, which will run the tasks required to setup
```

File: robocat/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+the `robocat` cluster. These "driver" cluster is the `dogfooding` cluster.
```
Comment (dibyom) [inline robocat/README.md]:
```suggestion
the `robocat` cluster. This "driver" cluster is currently the `dogfooding` cluster.
```

File: robocat/README.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+# Wait... or not
+
+Almost everything else is setup automatically via cronjobs scheduled in the
+`dogfooding` cluster. Since the setup of the DNS entry and the creation of the
+`ClusterIssuer` at the right time are still not automated, it's best for now
+to run through the setup "manually" by triggering the various cronjobs
+one by one, at least for the initial setup.
+Future hanges to the resources will be deployed nightly from git.
+
+# Point your cluster config to `dogfooding`
```
Comment (dibyom) [inline robocat/README.md]: My understanding is that this is part of running the CronJobs manually so second level heading  makes more sense 
```suggestion
## Point your cluster config to `dogfooding`
```

File: robocat/README.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+From this point on, most of the work will be done on the `dogfooding` cluster,
+so switch your configuration to point to it:
+
+```
+kubectl config use-context gke_tekton-releases_us-central1-a_dogfooding
+```
+
+# Run the cronjobs
```
Comment (dibyom) [inline robocat/README.md]:
```suggestion
## Run the cronjobs
```

File: robocat/README.md (hunk 1 above)
Comment (afrittoli) [reply robocat/README.md:5]: Done

File: robocat/README.md (hunk 2 above)
Comment (afrittoli) [reply robocat/README.md:6]: Done

File: robocat/README.md (hunk 4 above)
Comment (afrittoli) [reply robocat/README.md:80]: Done

File: robocat/README.md (hunk 3 above)
Comment (afrittoli) [reply robocat/README.md:71]: Done

**Pull request merged**